docs/guides/notebooks/configuration/groups.rst

```diff
@@ -0,0 +1,129 @@
+############################
+User groups for file sharing
+############################
+
+Create and manage closed user groups in order to share private files in the ``/project`` directory.
+
+.. _user-group-create:
+
+Create a closed group
+=====================
+
+Comanage webpage
+----------------
+
+* In a browser, navigate to `id.lsst.cloud <https://id.lsst.cloud>`_ and log in.
+
+* In the left menu sidebar, click on "Groups" and then "My Groups".
+
+* At right, a list of all the groups associated with your account will appear.
+
+* In the top horizontal menu, click on "Add Group".
+
+* Set the group properties:
+
+  * The group name should be short and must start with ``g_``.
+
+  * Write a short description.
+
+  * Leave the status as the default "Active".
+
+  * Leave the checkbox next to "Open" unselected, to create a closed group.
+
+  * Leave the checkbox next to "Require All for Nested Memberships" unchecked; this can be changed later if nested sub-groups are created.
+
+* Click the blue "Add" button.
+
+
+The next time you enter the Notebook Aspect, this group will be accessible.
+
+
+Command line
+------------
+
+At this time, it is not possible to create new groups from the terminal command line in the Notebook Aspect.
+
+It is possible to list all groups with ``getent group``, and to see all groups a
+user belongs to with ``groups <username>`` (see your own username with ``whoami``).
+
+
+Manage group membership
+=======================
+
+Only group owners can manage group membership.
+
+Comanage webpage
+----------------
+
+* In a browser, navigate to `id.lsst.cloud <https://id.lsst.cloud>`_ and log in.
+* In the left menu sidebar, click on "Groups" and then "My Groups".
+* At right, a list of all the groups associated with your account will appear.
+* In the "Name" column, click on the group name you want to add members to.
+* From the "Group Properties" page, select the tab "Members".
+* At upper right, use the "Add member" box to find and add group members.
+* In the "Permissions" column, make other group members "Owners" using the check boxes.
+
+
+Command line
+------------
+
+At this time, it is not possible to add users to groups from the terminal command line in the Notebook Aspect.
```

Comment:
Same comment as above. This will never be the case, remove "at this time".

Reply:
Yup also changed.